nixos/modules/system/boot/systemd/journald.nix

```diff
@@ -116,6 +134,11 @@ in
         "syslog.socket"
       ];
 
+    systemd.sockets.systemd-journald-audit.wantedBy = [
```

Comment:
So upstream systemd allows this to be enabled/disabled with in `.preset` and defaults to enabled. I guess we could mimmick that too here but I don't see harm in always enabling it?

Reply:
OH apparently we conditionalize the existence of this socket with `(!config.boot.isContainer)` further up.   so we have to have that condition here too I guess.

Personally not a big fan of this. The socket already has a `ConditionSecurity=audit`  in it so it won't start in containers; and  not shipping the unit file to save 400 bytes sounds like pre-mature optimization